README.md

```diff
@@ -51,7 +51,14 @@ the variable pizza inside your method.
 
 Your Answer:
 ```js
-// write code here
+
+  var pizza = {
+  temperature: "70",
+  bake: function () {
+      this.temperature = "300"
+  }
+}
```

Comment:
You have this concept down.  We recommend you use numbers (Integers) instead of Strings here.